### DISCUSSION TITLE
Dave (UK)

### DISCUSSION THREAD

**Discussion by ghost** [iptv-org/community#801] [Q&A]
### Channel Name

Dave

### Channel ID

Dave.uk

### Website

https://dave.uktv.co.uk/

https://uktvplay.uktv.co.uk/channel/dave

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by trancer1994** [comment]
https://live-uktv.simplestreamcdn.com/live2/uktvdave/bitrate1.isml/live.mpd

**Comment by ghost** [comment]
> https://live-uktv.simplestreamcdn.com/live2/uktvdave/bitrate1.isml/live.mpd

Thank you, @trancer1994. 

I'm getting my hopes up now but ... I don't know how to use that .mpd link?

I've tried with VLC Player but I'm obviously doing something wrong. 

It doesn't seem like I can do 

`Media>Open Network Stream>Paste: https://live-uktv.simplestreamcdn.com/live2/uktvdave/bitrate1.isml/live.mpd>Play`

Thanks again, very kind of you to help. Can you explain what I'm missing please.

**Comment by BellezaEmporium** [comment]
> https://live-uktv.simplestreamcdn.com/live2/uktvdave/bitrate1.isml/live.mpd

Needs license URL to work, https://sctoken.uktvapi.co.uk/?stream_id=733 gives a token and it's expiry.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19786
